### PR TITLE
Translate strings in *.ui correctly + typography guidelines

### PR DESCRIPTION
**Pull request by rluzynski**
There are 2 commits in this pull request, feel free to treat them separately.

### CODE REVIEW

**Comment by trapas** [comment]
Can one of the admins verify this patch?

File: data/ui/oops-menus.ui [hunk 1]
```diff
@@ -17,11 +17,11 @@
         <attribute name="action">win.details</attribute>
       </item>
       <item>
-        <attribute name="label" translatable="yes">_Open problem's data directory</attribute>
+        <attribute name="label" translatable="yes">_Open problem’s data directory</attribute>
         <attribute name="action">win.open-directory</attribute>
       </item>
       <item>
-        <attribute name="label" translatable="yes">_Copy problem's ID to Clipboard</attribute>
+        <attribute name="label" translatable="yes">_Copy problem’s ID to Clipboard</attribute>
```
Comment (ernestask) [inline data/ui/oops-menus.ui]: To be honest, the possessive feels weird here altogether (and the capitalization of “clipboard”). “Copy problem ID to clipboard” sounds way better.

Comment (ernestask) [reply data/ui/oops-menus.ui]: (Same with “problem’s data” above)

File: src/gnome_abrt/dbus_problems.py [hunk 2]
```diff
@@ -148,7 +148,7 @@ def chown_problem(self, problem_id):
             return True
         except dbus.exceptions.DBusException as ex:
             logging.warning(
-                    _("Can't chown problem '{0}' over DBus service: {1!s}")
+                    _("Can’t chown problem “{0}” over D-Bus service: {1!s}")
```
Comment (ernestask) [inline src/gnome_abrt/dbus_problems.py]: Just commenting: this string doesn’t sound easily translatable, as it uses “chown” as a verb.

File: src/gnome_abrt/directory_problems.py [hunk 3]
```diff
@@ -213,7 +213,7 @@ def _disable_on_max_watches(self, ex, directory):
         logging.debug("Could not add inotify for directory '{0}': '{1}'"
                             .format(directory, ex))
         logging.warning(
-_("You have probably reached inotify's limit on the number of watches in '{0}'."
+_("You have probably reached inotify’s limit on the number of watches in “{0}”."
```
Comment (ernestask) [inline src/gnome_abrt/directory_problems.py]: “You have probably reached the inotify limit”

**Comment by ernestask** [comment]
I’m fine with more Unicode and all that jazz.

**Comment by ernestask** [comment]
@piotrdrag, maybe you have an opinion?

**Comment by piotrdrag** [comment]
I like Rafał’s changes, but also agree with your notes (still not a native speaker, though.)

**Comment by piotrdrag** [comment]
By the way, some of the strings look like they go straight to the syslog — I thought we’re generally not translating these?

**Comment by ernestask** [comment]
> By the way, some of the strings look like they go straight to the syslog — I thought we’re generally not translating these?

Good point, we should not do that. They’re not very helpful translated, anyway.

@rluzynski, can you unmark those strings?

**Comment by rluzynski** [comment]
Done:

* the strings going to the log unmarked from translation,
* ```"problem's"``` reworded to ```"problem"``` (actually: ```"the problem"```),
* ```"inotify's"``` reworded to ```"the inotify"```.

It turned out that there are no more apostrophes or quotes to convert to the fancy Unicode characters.

**Comment by ernestask** [comment]
Thank you, @rluzynski.

**Pull request merged**